BookTracking.Web/src/pages/Home/ExpandedBookRow.tsx

```diff
@@ -0,0 +1,22 @@
+﻿import { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+import { Anchor, NavLink, Table } from '@mantine/core';
+import { Book } from '../../models/Book';
+
+interface ExpandedBookRowProps {
+  book: Book;
+}
+
+export const ExpandedBookRow = ({ book }: ExpandedBookRowProps): ReactNode => {
+  return (
+    <Table>
+      <Table.Thead>
+        <Table.Th>Author</Table.Th>
```

Comment:
Only `Table.Tr` are valid direct children of `Table.Th` components. All html tables need `tr` as direct children of `thead` and `tbody` elements which is what the underlying elements of these Mantine components are.